### DISCUSSION TITLE
除了元信息,是否有其他方法可以阻止未授权的用户通过构造/d/链接下载不属于他的文件

### DISCUSSION THREAD

**Discussion by justghostof** [AlistGo/alist#4221] [Q&A]
我注意到使用本地储存时任何用户只需知道文件名就可以直接通过/d链接下载任意一个文件，即使这不在他的基本路径
下，生成的/d/链接似乎除了元信息没有任何鉴权手段，这可能导致安全问题，alist是否有提供其他方法保护某些可能的隐私文件

**Comment by Rektyfikowany** [comment]
当前主要通过[签名所有](https://alist.nn.ci/zh/config/global.html#%E7%AD%BE%E5%90%8D%E6%89%80%E6%9C%89)配合元信息来实现鉴权，如有更好方案欢迎提出
